gcm/error.go

```diff
@@ -0,0 +1,32 @@
+package gcm
+
+type GCMErrorResponseCode int
```

Comment:
That code will be used at `supervisor.go` when to receive response from GCM, and I would like to use `stringer` because I want to check Const name as error response message the following:

```go
switch resp.Err {
case gcm.InvalidRegistration.String():
  // handling error
case gcm.MissingRegistration.String():
  // ...
}
```

gcm response example:
```json
{
    "multicast_id": 6100981481952134000,
    "success": 3,
    "failure": 1,
    "canonical_ids": 0,
    "results": [
      {
        "message_id": "0:1487233455306332%caa8591dcaa8591d"
      },
      {
        "message_id": "0:1487233455306303%caa8591dcaa8591d"
      },
      {
        "message_id": "0:1487233455307376%caa8591dcaa8591d"
      },
      {
        "error": "InvalidRegistration"
      }
    ]
  }
```

One of `results` array is each GCM response result and `error` value is filled at `resp.Err`(string type).